packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -357,6 +357,7 @@ export class SharedTree
  * @privateRemarks This function will be removed if/when the branching API becomes public,
  * but it (or something like it) is necessary in the meantime to prevent the alpha types from being exposed as public.
  * @alpha
+ * @deprecated This API is superseded by {@link TreeContext}, which should be used instead.
```

Comment:
Should we file an item so we remember to remove these?

Reply:
Great idea, I filed 22595